### ISSUE TITLE
Slack Link Incorrect in README

### ISSUE THREAD

**Issue by MonideepDe** [astronomer/astronomer-cosmos#387]
Hi there!

The slack channel link mentioned in [README.rst](https://github.com/astronomer/astronomer-cosmos/blob/223e4bf3995d6aa23b521542fad0359566eff6bc/README.rst) in incorrect.

Clicking on the [link](https://apache-airflow.slack.com/join/shared_invite/zt-1vo1rxgc3-EfsbdDzqrV51fddWOc6GOQ#/shared-invite/error) results in below error:
```
This link is no longer active
To join this workspace, you’ll need to ask the person who originally invited you for a new link.
```

Can someone share the right link? and in the process we can get correct the documentation :-) 

Thanks!

**Comment by harels** [comment]
Looks like the slack link expired, will take care of this

**Comment by harels** [comment]
this was resolved by #394
